brooklyn-tosca-transformer/src/main/java/io/cloudsoft/tosca/a4c/brooklyn/Alien4CloudFacade.java

```diff
@@ -268,27 +291,49 @@ public boolean isDerivedFrom(String nodeId, Alien4CloudApplication toscaApplicat
         NodeTemplate nodeTemplate = toscaApplication.getNodeTemplate(nodeId);
         IndexedArtifactToscaElement indexedNodeTemplate = getIndexedNodeTemplate(nodeId, toscaApplication);
 
-        Optional<Interface> optionalIndexedNodeTemplateInterface = NodeTemplates.findInterfaceOfNodeTemplate(
-                indexedNodeTemplate.getInterfaces(), VALID_INTERFACE_NAMES);
+        operations.putAll(getInterfaceOperationsMap(indexedNodeTemplate, VALID_INTERFACE_NAMES));
 
         Optional<Interface> optionalNodeTemplateInterface = NodeTemplates.findInterfaceOfNodeTemplate(
                 nodeTemplate.getInterfaces(), VALID_INTERFACE_NAMES);
 
-        if (optionalIndexedNodeTemplateInterface.isPresent()) {
-            operations = MutableMap.copyOf(optionalIndexedNodeTemplateInterface.get().getOperations());
-        }
-
         if (optionalNodeTemplateInterface.isPresent()) {
             operations.putAll(optionalNodeTemplateInterface.get().getOperations());
         }
         return operations;
     }
 
+    private Map<String, Operation> getConfigureInterfaceOperationsMap(String nodeId, Alien4CloudApplication toscaApplication, String requirementId) {
+        Map<String, Operation> operations = MutableMap.of();
+        Optional<IndexedArtifactToscaElement> indexedRelationshipTemplate = getIndexedRelationshipTemplate(nodeId, toscaApplication, requirementId);
+        if(!indexedRelationshipTemplate.isPresent()){
```

Comment:
Minor, consistency: you check the opposite condition here and return a copy of the map in `getInterfaceOperationsMap`. Use the same pattern for both.